ClientRuntimes/CSharp/Microsoft.Rest.ClientRuntime/project.json

```diff
@@ -24,6 +24,9 @@
         "System.Net.Http": "",
         "System.Net.Http.WebRequest": "",
         "System.Xml": ""
+      },
+      "dependencies": {
+        "Newtonsoft.Json": "6.0.8"
```

Comment:
This seems like a step backwards. What's the plan for eventually bumping this to 7.x? We can't stay on this version forever.